contract/src/burn/tests.rs

```diff
@@ -45,22 +43,20 @@ fn burn_token_by_oracle_for_valid_owner() {
 }
 
 #[test]
-#[should_panic(expected = "Account doesnt own the token")]
+#[should_panic(expected = "Account doesn't own the token")]
 fn burn_token_by_oracle_for_invalid_owner() {
     let oracle = oracle();
     let mut context = Context::new(&oracle);
 
-    let reference_booster_data = BalanceBoosterData {
-        media: "bafkiufjkssmvjby7srbr44ivexs74937vjby7hhoobl3q3twim3bgxffrm".to_string(),
-        media_hash: Base64VecU8::from(b"w5SZrIcpoakfcRUl5EvCqerjZzscFe4bnZDNhNcpYs=".to_vec()),
-        denomination: U128(2_000_000),
-    };
+    let media = "bafkiufjkssmvjby7srbr44ivexs74937vjby7hhoobl3q3twim3bgxffrm".to_string();
+    let media_hash = Base64VecU8::from(b"w5SZrIcpoakfcRUl5EvCqerjZzscFe4bnZDNhNcpYs=".to_vec());
```

Comment:
These hashes are duplicated in many places. Maybe extract them to variables?

Reply:
They are slightly different :)